packages/skills/weather_station_ledger/handlers.py

```diff
@@ -229,9 +229,8 @@ def _handle_inform(self, msg: FIPAMessage, dialogue: Dialogue) -> None:
             logger.info("[{}]: checking whether transaction={} has been received ...".format(self.context.agent_name,
                                                                                              tx_digest))
             proposal = cast(Description, dialogue.proposal)
-            total_price = cast(int, proposal.values.get("price"))
             ledger_id = cast(str, proposal.values.get("ledger_id"))
-            is_settled = self.context.ledger_apis.is_tx_settled(ledger_id, tx_digest, total_price)
+            is_settled = self.context.ledger_apis.is_tx_settled(ledger_id, tx_digest)
```

Comment:
can we add a `TODO: check the tx_digest references a transaction with the correct terms`?